README.md

```diff
@@ -20,6 +20,31 @@ kashmir
 [npm-image]: https://badge.fury.io/js/generator-kashmir.svg
 [npm-url]: https://npmjs.org/package/generator-kashmir
 
+
+## Build commands
+---
+
+### Compile theme
+
+In order to compile the assets of the theme, this also compiles sass inside components,
+
+```bash
+gulp
```

Comment:
@taher-pro This should be `yarn gulp` so that there is no dependency on `gulp-cli`

Reply:
@skippednote Ok Noted. I will add this change with the patternlab docs on a new MR.